serenity.cpp

```diff
@@ -255,15 +255,12 @@ class Frontend : public GUI::Widget {
         return midend_wants_statusbar(m_midend);
     }
 
-    void set_statusbar(GUI::Statusbar* statusbar) {
-        m_statusbar = statusbar;
+    void set_statusbar(NonnullRefPtr<GUI::Statusbar> statusbar) {
+        m_statusbar = statusbar.ptr();
```

Comment:
`m_statusbar` should remain `RefPtr<T>` storing a recounted object locally without a refcount is a recipe for disaster.